man/man1/git-town.1

```diff
@@ -72,3 +74,9 @@ Add a new perennial branch
 
 .IP "git town perennial-branches --remove <branch_name>" 4
 Remove branch from perennial branches
+
+.IP "git town pull-branch-strategy" 4
+Displays your pull branch strategy
```

Comment:
I don't think the first current is needed. Also none of the other commands reference the repository. The readme mentions git town is configured per repository. We could add something like that to this man page.